lib/openxml/elements/run_properties.rb

```diff
@@ -39,6 +43,10 @@ def initialize(*boolean_properties, **value_properties)
         end
       end
 
+      def font_color=(color)
+        self.push(OpenXml::Elements::RGBColor.new(color))
+      end
```

Comment:
The other option is to have the value override any previous setting. The thing about that and not reusing is that you could change the value out from under another slide that is using the same text object. I don't think that is a good idea.